change/@fluentui-react-spinner-fc7ce5f3-977b-48c9-883e-6fe81d838968.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30330&commitId=825ca1c87331ada5dba8a992d0d61988ba6dd029&env=prod&clientType=FLUENTUI)


  <details><summary><b>Spinner converged</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Primary + size.chromium.png|2873| Changed|
|Spinner converged.Primary - High Contrast.chromium.png|9291| Changed|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|46656| Changed|
|Spinner converged.Primary with Label - RTL.chromium.png|1518| Changed|
|Spinner converged.Inverted.chromium.png|6158| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|6158| Changed|
|Spinner converged.Inverted - High Contrast.chromium.png|9487| Changed|
|Spinner converged.Inverted with Label.chromium.png|36215| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|55400| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|55794| Changed|
|Spinner converged.Primary with Label.chromium.png|1518| Changed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|36215| Changed|
|Spinner converged.Primary.chromium.png|182| Changed|
|Spinner converged.Primary - Dark Mode.chromium.png|7916| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>